src/View3d.ts

```diff
@@ -218,6 +218,7 @@ export class View3d {
 
   // channels is an array of channel indices for which new data just arrived.
   onVolumeData(volume: Volume, channels: number[]): void {
+    this.image?.setScale(volume.scale, volume.contentSize, volume.contentOffset);
```

Comment:
`setScale` is currently both a member of `View3d`'s public interface and is called to set the scale of the volume. The volume internally believes that it is a 1x1x1 cube, but is scaled such that its maximum dimension is 1 and its other dimensions are an appropriate ratio. This scale value is stored both as a property of `Volume` and a property of `VolumeRenderSettings`.

It's not clear to me that an external user would see what they expect if they called `setScale`. E.g., if they tried to scale z by 0.5, and the volume were less than half as large in z as in some other dimension, the volume would actually grow in z. Additionally, `VolumeRenderImpl`s use a hodgepodge of reading `scale` from settings vs. reading directly from the volume, and I've just made this worse by following the same discipline for `contentSize` and `contentOffset`.

I think it would make the most sense to consider the `scale` in settings to be a separate user-defined scale factor which is never modified internally. `VolumeRenderImpl`s would always read `Volume` scale directly from the `Volume` object, then additionally scale by this setting. If `Volume` dimensions update (which they never did before this PR), we'd use a separate pathway to update the `VolumeRenderImpl` that doesn't go through `updateSettings` at all. Does this check out? Or could `setScale` even go away completely?

Reply:
`setScale` can go away completely as long as `setVoxelSize` remains exposed and works sensibly